CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ CHANGELOG
 - Fix exporting of cluster logs when there is no prefix specified, previously exported to a `None` prefix.
 - Fix rollback not being performed in case of cluster update failure.  
 - Fix RootVolume schema for the HeadNode.
+- Fix EfaSecurityGroupValidator. Previously, it may produce false failures when custom security groups were provided and EFA was enabled.
```

Comment:
What about something like: `Fix EfaSecurityGroupValidator to correctly validate custom Security Groups with more than 1 UserIdGroupPairs when EFA is enabled.`

Reply:
I tried to use more user-friendly wordings in the change log, and more technical wordings in the commit description. But I am ok to change the message here